### PR TITLE
Ui bug fixes

### PR DESCRIPTION
**Pull request by katibest**
resolves: #240 #236 #237

### CODE REVIEW

File: apps/explorer_web/assets/css/_typography.scss [hunk 1]
```diff
@@ -41,12 +41,12 @@ p {
 
 a {
   text-decoration: none;
-  color: $gray-800;
-  font-size: 12px;
+  color: $primary;
```
Comment (acravenho) [inline apps/explorer_web/assets/css/_typography.scss]: @katibest Will this change the color of the menu links as well?

Comment (acravenho) [reply apps/explorer_web/assets/css/_typography.scss]: Also, the `font-size` of `12px` is `1px` small than the rest of the text.

Comment (katibest) [reply apps/explorer_web/assets/css/_typography.scss]: Good catch on the menu links - Ryan and I missed that!
As for the font-size it's 13px, which is the same as everything else ( see the next line).

**Comment by tmecklem** [comment]
@KronicDeth can you take a look at the test failure on this one? All the changes in this PR are on explorer_web, but the test failure is in explorer:

```
  1) test run/1 allows tasks to be programmatically retried (Explorer.BufferedTaskTest)
     test/explorer/buffered_task_test.exs:129
     No message matching {:run, {0, [1, 2]}} after 100ms.
     The process mailbox is empty.
     code: assert_receive {:run, {0, [1, 2]}}
     stacktrace:
       test/explorer/buffered_task_test.exs:134: (test)
```

**Comment by KronicDeth** [comment]
@tmecklem it is a flaky test, I just use the "Rerun failed jobs" option in CircleCI when it happens.  I have done that with your build.

To do this on your own in the future.

1. Click one of the [Details](https://circleci.com/gh/poanetwork/poa-explorer/6205?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) links here in GitHub
2. Click the [primary](https://circleci.com/gh/poanetwork/poa-explorer/6205?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) Workflow link
   screen shot 2018-06-07 at 1 45 06 pm
3. Click the down chevron on the Rerun menu
     screen shot 2018-06-07 at 1 45 39 pm
4. Click "Rerun failed jobs"

A new workflow with all the successful job from the previous workflow + a rerun of the failed jobs will be started 

screen shot 2018-06-07 at 1 46 48 pm


If you click on the branch name link

screen shot 2018-06-07 at 1 46 48 pm

 it will go to the Workflow view

screen shot 2018-06-07 at 1 47 43 pm


Where you can watch all the parallel builds.  CircleCI's integration with GitHub will automatically update the statuses correctly if the new job succeeds.

**Comment by jimmay5469** [comment]
Thanks @KronicDeth!

**Pull request merged**